package.json

```diff
@@ -18,7 +18,7 @@
     "styled-components": "^5.3.6"
   },
   "scripts": {
-    "start": "PORT=3666 react-scripts start",
```

Comment:
revert this change. add the port before commiting.

Add back the port. You may remove the port to run on local

Reply:
done